File: typescript/src/calisthenics-exercises/dont-use-abbreviations.ts
export class C {
  u: string
  p: string
  s: string
  e: 'dev' | 'prod'

  constructor(u: string, p: string, s: string, e: 'dev' | 'prod') {
    this.u = u
    this.p = p
    this.s = s
    this.e = e
  }

  cnx(): string {
    return `${this.u}:${this.p}@${this.s}/${this.e}`
  }
}

File: csharp/src/calisthenics-exercises/DontUseAbbreviations.cs
namespace RefactoringAvanzado.CalisthenicExercises;

public class C
{
    public string U { get; set; }
    public string P { get; set; }
    public string S { get; set; }
    public string E { get; set; }

    public C(string u, string p, string s, string e)
    {
        U = u;
        P = p;
        S = s;
        E = e;
    }

    public string Cnx()
    {
        return $"{U}:{P}@{S}/{E}";
    }
}
